java/client/src/main/java/glide/api/commands/GeospatialIndicesBaseCommands.java

```diff
@@ -78,4 +80,58 @@ CompletableFuture<Long> geoadd(
      * }</pre>
      */
     CompletableFuture<Double[][]> geopos(String key, String[] members);
+
+    /**
+     * Returns the distance between <code>member1</code> and <code>member2</code> in the geospatial
```

Comment:
Please reword (up to you)
```suggestion
     * Returns the distance between <code>member1</code> and <code>member2</code> saved in the geospatial
```
Otherwise it could be read as 'distance ... in the index'